lib/beacon_api/controllers/v2/beacon_controller.ex

```diff
@@ -36,7 +36,7 @@ defmodule BeaconApi.V2.BeaconController do
 
   def get_block(conn, %{block_id: "0x" <> hex_block_id}) do
     with {:ok, block_root} <- Base.decode16(hex_block_id, case: :mixed),
-         block <- Blocks.get_signed_block(block_root) do
+         %{} = block <- Blocks.get_signed_block(block_root) do
```

Comment:
?

Reply:
We found this with @avilagaston9. The result can be either a map or `nil`, but we aren't matching on the map. This can result in weird OK responses.